tests/test_check_django_migrations_filename.py

```diff
@@ -19,11 +19,28 @@ def hook():
         'houses/migrations/0004_auto_20190315_0043.py',
     ]
 )
-def test_migration_filename_validate(filename, capsys, hook):
+def test_migration_filename_invalid(filename, capsys, hook):
     result = hook.validate(filename)
 
     output, _ = capsys.readouterr()
 
     assert filename in output
     assert 'invalid migration filename' in output
     assert result is False
+
+
+@pytest.mark.parametrize(
+    'filename',
+    [
+        'core/migrations/0002_CreateUserAdmin.py',
```

Comment:
Utilizamos o padrão snake_case ou underscore_case para nomes de migrations, acho que precisa ajustar o teste para refletir o correto. Nunca alguém fez o nome das migrations usando o estilo CamelCase, mas fiquei pensando se vale a pena ajustar o hook para esses casos também.. :thinking: Como já aprovei, e o objetivo do hook já está alcançado, penso que esse ajuste pode ser feito em outro PR, se for relmente necessário. Aguardo a opinião do @daneoshiga e o @lamenezes também sobre isso :+1:

Reply:
acho uma boa garantir que o nome do arquivo está em camel case sim. bem lembrado @bratomes. talvez tenha alguma função no olist commons [1] que possa ajudar nessa tarefa

[1] https://github.com/olist/olist-commons/blob/master/olist_commons/converters.py

Reply:
Creio que isso deveria ser outro hook, visto que esse projeto não é especifico para o Olist, caso não seja uma convenção. Deste modo não travaríamos o nosso padrão para outros usuários.

Reply:
vale lembrar que o arquivo gerado pela migração é um módulo python e existe sim um padrão da linguagem para nomear módulos que é usar snake case e não camel case e outros.

Reply:
Mas este hook aborda o nome das migrations, por isso acho que faz sentido incluir esse tratamento. Caso não utilizem o mesmo padrão que a Olist, em nomes de migrations, basta não utilizar o hook.